tests/agent_server/deploy/scheduler_test_util.py

```diff
@@ -68,7 +68,7 @@ def convert_resource(resource: JsonType) -> ResourceDetails:
     m.update(character.encode("utf-8"))
     attribute_hash = m.hexdigest()
 
-    return ResourceDetails(out, attribute_hash)
+    return ResourceDetails(attributes=out, attribute_hash=attribute_hash)
```

Comment:
oh, now I understand. The resource details `attribute` field doesn't just contain the attributes, but the serialized resource? Then I think the field should have a different name? 

or this is a mistaken update of the testcase, and it should be
```suggestion
    return ResourceDetails(attributes=attributes, attribute_hash=attribute_hash)
```

Reply:
Definitely a mistaken update, thanks.